src/Rootfs.jl

```diff
@@ -749,6 +749,10 @@ function expand_gfortran_versions(platform::AbstractPlatform)
         return [platform]
     end
 
+    if sanitize(platform) == "memory"
+        return platform #MSAN can't use libgfortran
```

Comment:
I'd say "doesn't use", rather than "can't use", it just uses a different fortran runtime library